flocker/node/agents/ebs.py

```diff
@@ -1181,8 +1185,7 @@ def attach_volume(self, blockdevice_id, attach_to):
         ignore_devices = pset([])
         for attach_attempt in range(3):
```

Comment:
If we're going to limit ourselves to attaching to local machine (a reasonable assumption for now) then we should throw an exception if `blockdevice_id != self.compute_instance_id()` saying attaching elsewhere is unsupported. Otherwise when the assumption is broken we'll get some weird behavior.